docs/source/theory.rst

```diff
@@ -0,0 +1,235 @@
+Theory
+======
+
+Introduction
+------------
+
+The Neffint package is mainly designed for with computing Fourier integrals, i.e.
+
+.. math::
+    \int_{-\infty}^{\infty} \psi(\omega) e^{i \omega t} d\omega,
+
+using a quadrature method largely based on section 1.6.3 in [N.Mounet-PhD]_, which builds on the work of [Filon]_.
```

Comment:
we might want to cite as well the less detailed version of the method published [here](https://cds.cern.ch/record/2841806/files/document.pdf)